tests/wpt/web-platform-tests/.gitignore

```diff
@@ -49,4 +49,4 @@ scratch
 /webaudio/idl/*
 
 # w3c-test.org PR-branch mirroring
-submissions/
+/submissions/
```

Comment:
hmm, I'm wondering if we should send a PR to upstream so that we won't need to keep updating this 🤔